### PR TITLE
adds plus, minus, times and by and removes default from inc/dec

### PR DESCRIPTION
**Pull request by stoeffel**
refs #34

### CODE REVIEW

**Comment by tomekwi** [comment]
:+1:

I’m just wondering how to name these functions. My [original proposal](https://github.com/stoeffel/1-liners/issues/1#issuecomment-97907478) (1) was inspired by Scala, where – AFAIK – things like `+` and `-` are themselves functions. Not sure though.

* `plus`
* `minus`
* `times` (I don’t like this name)
* `by`

(2) if we go down the trail of `sum`:

* `sum`
* `difference`
* `multiplication`
* `division`

And there’s (3) my current favorite:

* `add`
* `substract`
* `multiply`
* `divide`

Do you guys know how these things are names in other languages?

**Comment by stoeffel** [comment]
I guess the third one is the most common one, but I like the scala one. Would look nice in compose.

```js
reduce(compose, [
  plus(3),
  times(2),
  by(10),
  minus(1)
);
```

File: tests/by.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { equal } from 'assert';
+import by from '../by';
+
+test('#by', () => equal(by(6, 2), 3));
```
Comment (stoeffel) [inline tests/by.js]: How about we flip the arguments to be more FP?

Comment (stoeffel) [inline tests/by.js]: actually `flip` is really simple lets keep the order of the args like they are.

**Comment by tomekwi** [comment]
:+1: :+1:

**Pull request merged**